src/vs/workbench/browser/parts/editor/editorCommands.ts

```diff
@@ -18,11 +18,18 @@ import { ICommandService } from 'vs/platform/commands/common/commands';
 import { IMessageService, Severity, CloseAction } from 'vs/platform/message/common/message';
 import { Action } from 'vs/base/common/actions';
 import { KeyMod, KeyCode } from 'vs/base/common/keyCodes';
+import { TPromise } from 'vs/base/common/winjs.base';
+
+export const CLOSE_UNMODIFIED_EDITORS_COMMAND_ID = 'workbench.command.closeUnmodifiedEditors';
+export const CLOSE_EDITORS_IN_GROUP_COMMAND_ID = 'workbench.command.closeEditorsInGroup';
+export const CLOSE_EDITOR_COMMAND_ID = 'workbench.command.closeActiveEditor';
+export const CLOSE_OTHER_EDITORS_IN_GROUP_COMMAND_ID = 'workbench.command.closeOtherEditors';
 
 export function setup(): void {
 	registerActiveEditorMoveCommand();
 	registerDiffEditorCommands();
 	registerOpenEditorAtIndexCommands();
+	registerExplorerCommands();
```

Comment:
@isidorn I would not call this explorer commands for layering reasons (explorer is a contribution to the workbench, and editor is not). Just call them editor commands or similar.

Reply:
Make sense,  I will change it.